release-process.md

```diff
@@ -223,23 +182,16 @@ $ git push apache v1.1.1
 
 The website repository is located at
 <a href="https://github.com/apache/spark-website">https://github.com/apache/spark-website</a>.
-Ensure the docs were generated with the PRODUCTION=1 environment variable.
 
 ```
-# Build the latest docs
-$ git checkout v1.1.1
-$ cd docs
-$ PRODUCTION=1 jekyll build
-
-# Copy the new documentation to Apache
+# copy the docs of the voted RC to spark-website
+$ svn co "https://dist.apache.org/repos/dist/dev/spark/v2.4.0-rc5-docs" spark-docs
```

Comment:
Since you may have this info locally, how many files are in the current docs build?

Last time I tried to checkout a huge directory from svn, the ASF infra guys blocked our network from talking to the ASF servers. So I'd avoid this if that directory is kinda large.

I don't remember if I actually did that or just thought about it, but one idea was to leave the docs build locally after the rc is generated, instead of cleaning it up when the docker script is gone. That would allow you to just upload your local copy to spark-website.

Reply:
For 2.4.0 it's 3572 files, 145 MB.

During a release, we upload these doc files many times(for each RC), and download them once. Will this be a problem?

Reply:
BTW during RC voting, if one people want to validate the doc, he has to check it out locally and start a server to look at the web pages. If the ASF infra blocks it, it will be hard for people to evaluate an RC.

Reply:
That's a lot of files. I think the thing the ASF doesn't like is a burst of requests, and this might trigger it. Uploading is not a problem, since it's authenticated. Downloading is not, so it could be a DoS, which is why they do the blocking.

> if one people want to validate the doc, he has to check it out locally 

Actually they can see it directly on the ASF server without downloading it locally. Using a browser and manually looking at the logs automatically "throttles" the number of requests, so it shouldn't trigger any warnings to the infra guys.

I think it's safer to recommend keeping the docs for the last RC locally in case it passes, and only checkout from svn if necessary.